app/models/katello/host/subscription_facet.rb

```diff
@@ -34,6 +35,19 @@ def import_database_attributes(consumer_params)
         end
       end
 
+      def update_consumer_type(consumer_params)
+        if consumer_params.try(:[], 'type').try(:[], 'label') == 'hypervisor'
+          self.consumer_type = 'hypervisor'
+        elsif consumer_params.try(:[], 'facts').try(:[], 'virt.is_guest') == "true"
```

Comment:
This conversation got deep into whether we include the virtual vs. physical nature of a host as part of this. That feels like it should be an automatic, separate change to all of this. And to keep this as focused on consumer type to allow users to find hypervisors and guests. I'd prefer to just open a separate issue for that ?